apps/condo/domains/billing/schema/resolvers/propertyResolver.js

```diff
@@ -49,9 +50,17 @@ class PropertyResolver extends Resolver {
         if (loaded) {
             return loaded
         }
+
+        const knownMissingProperty = this.missingOrganizationPropertiesIndex[addressKey]
+        if (knownMissingProperty) {
+            return loaded
```

Comment:
It's ok that if we check not existed organization property, then create this property and recheck `getOrganizationProperty`, then it returns that this property does not exist?

Reply:
This logic are working in other way. Firstly it check all properties, calc missing, create missing. For case, when some property are omit - it going to check and check the same property again and again. So, if we already know that such property are omit - why we have to check it again?